Libplanet/Net/Swarm.cs

```diff
@@ -45,7 +45,7 @@ public class Swarm<T> : IDisposable
         private CancellationTokenSource _workerCancellationTokenSource;
         private CancellationToken _cancellationToken;
 
-        private (long, BoundPeer, HashDigest<SHA256>)? _demandBlockHash;
+        private (BlockHeader, BoundPeer, HashDigest<SHA256>)? _demandBlockHash;
```

Comment:
Maybe it is better to make it a separate type... 🤔

Reply:
Do you mean by tuple? Or `BlockHeader`?

Reply:
Tuple. 😅